web/src/Pages/Login/index.tsx

```diff
@@ -21,18 +34,35 @@ const Login = () => {
             base: '75%',
             md: '25%',
           }}
+          value={senha}
+          onChange={(evento: any) => setSenha(evento.target.value)}
           type="password"
           placeholder="password"
         />
       </View>
       <View style={styles.buttonSBS}>
         <View style={styles.configButtonRegister}>
-          <Button style={styles.buttonRegister}>Login</Button>
+          <Button
+            style={styles.buttonRegister}
+            onPress={async () => {
+              const JWT = await login(email, senha);
+
+              if (JWT) {
+                setJwt(JWT);
+                navigate('/account');
+              } else {
+                //aqui altera o estado do erro
+                setErro(true);
+              }
+            }}>
+            Login
+          </Button>
         </View>
         <View style={styles.configButtonCancel}>
           <Button style={styles.buttonCancel}>Cancel</Button>
         </View>
       </View>
+      {erro == true ? <Text>Algo deu errado, tente novamente</Text> : undefined}
```

Comment:
Instead of `erro == true` you can just use `erro` as the condition, and instead of a ternary if it is possible to use the logical operator `&&`, like this: `erro && <Text>...`